### PR TITLE
Removed paths from workflows

### PR DESCRIPTION
**Pull request by Marcin-Radecki**
# Description

This PR removed `paths` from workflows, so now all checks are triggered unconditionally. This has below motivations:
* we've had an issue that checks excluded packages broke due to one PR slipped change that updated excluded package build, as this workflow has bug which does not trigger it when `relations`  
* build and push cliain worfklow has bug which does not trigger on some paths, e.g. when `aleph-client` is changed

This PR also removes `matrix` from check excluded packages as now we run N times check excluded packages, we don't need that matrix anyway as single workflow run takes now 8 minutes. 

## Type of change

Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)

# Checklist:

### CODE REVIEW

File: .github/workflows/build-and-push-cliain.yml [hunk 1]
```diff
@@ -6,12 +6,7 @@ on:
     branches:
       - 'main'
     paths:
```
Comment (obrok) [inline .github/workflows/build-and-push-cliain.yml:8]:
```suggestion
```

Comment (deuszx) [inline .github/workflows/build-and-push-cliain.yml:8]: Why not remove the `paths` entry entirely?

Comment (Marcin-Radecki) [reply .github/workflows/build-and-push-cliain.yml:8]: Right, will remove

**Review by obrok — APPROVED** [review]
I'm not a fan of `paths`, because I think it's too easy to make a bug (like not building when a dependency changes), so +1 from me. I prefer the `matrix` approach even when the builds are fast, but it does have some drawbacks too, so no objections from me there.

File: .github/workflows/build-and-push-cliain.yml (hunk 1 above)
Comment (Marcin-Radecki) [reply .github/workflows/build-and-push-cliain.yml]: done

Comment (Marcin-Radecki) [reply .github/workflows/build-and-push-cliain.yml]: done

**Review by deuszx — APPROVED** [review]

**Pull request merged**